presto-main/src/main/java/com/facebook/presto/operator/ConnectorOutputMetadataWrapper.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator;
+
+import com.facebook.presto.spi.connector.ConnectorOutputMetadata;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import static java.util.Objects.requireNonNull;
+
+public class ConnectorOutputMetadataWrapper
```

Comment:
Hiding a problem doesn't solve the problem.

I see your comment about this can be a list of `List<String>`: https://github.com/prestodb/presto/pull/13528#discussion_r333299827. Curious about the decision consideration behind it. Let's quickly talk in person about this.